build/buildPackage.sh

```diff
@@ -127,6 +126,15 @@ manifest="
 
 "
 
+if [ "$#" -lt "1" ]; then
```

Comment:
> It's the packaging of IE modules done correctly if renderman doesn't exist? Then the cortex scons build doesn't build the lib & modules and the package looks for IECore* so it handles this correctly?

Yeah, if they're not built then they won't be packaged. But if they _are_ built, and then you run `buildPackage.sh` requesting no renderman, you'll end up with a package with the IECoreRI module but with no renderMan directory.

Maybe what you actually want to do is to package the renderMan directory if it's there, and not package it if it isn't? Do we need the command line flag at all?